Project.toml

```diff
@@ -16,6 +16,11 @@ julia = "^1.6.0"
 
 [extras]
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+MadNLP = "2621e9c9-9eb4-46b1-8089-e8c72242dfb6"
+NLPModelsIpopt = "f4238b75-b362-5c4c-b852-0801c9a21d71"
+JuMP = "4076af6c-e467-56ae-b986-b466b2749572"
+Ipopt = "b6b21f68-93f8-5de0-b562-5493be1d77c9"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 
 [targets]
-test = ["Test"]
\ No newline at end of file
+test = ["Test", "MadNLP", "NLPModelsIpopt", "JuMP", "Ipopt", "Random"]
```

Comment:
LinearAlgebra here

Reply:
I originally had that in there, but I the test failed (see [link](https://github.com/MadNLP/DynamicNLPModels.jl/runs/6838304172?check_suite_focus=true))

Reply:
ah, yes, you're right. no need for the change